src/Containers/ReqDetailContainer.styles.scss

```diff
@@ -1,8 +1,22 @@
 @import "./../styles/variables";
 
+$close-button-width: 35px;
+
 .req-detail-container {
   flex: 1 1 auto;
   height: 100%;
   overflow: auto;
   background: $white-95;
+  position: relative;
+}
+
+.nav-tabs {
+  padding-left: $close-button-width;
+}
+
+.close-button {
+  position: absolute;
+  left: $gutter / 2;
+  top: $gutter / 2;
```

Comment:
i might have missed this on earlier pr's but should we switch to $size variables instead of $gutter to stay consistent with the sl ui?

`
// SIZES
$size-base: 4px;

$size-xs: $size-base; // 4px;
$size-s: $size-base * 3; // 12px;
$size-m: $size-base * 4; // 16px;
$size-l: $size-base * 5; // 20px;
$size-xl: $size-base * 7; // 28px;
$size-xxl: $size-base * 9; // 36px;
$size-xxxl: $size-base * 11; // 44px;`

Reply:
Good point, I missed that.

Reply:
I will handle this in separate PR.